includes/Ad_Manager.php

```diff
@@ -77,15 +77,21 @@ public function print_ad_manager_tag() {
 		if ( ! $enabled || ! $slot ) {
 			return;
 		}
+
+		$application_json = [];
+
+		$ad_attributes = [];
+		$ad_attributes[ 'type' ] = 'doubleclick';
+		$ad_attributes[ 'data-slot' ] = $slot;
+
+		$application_json['ad-attributes'] = $ad_attributes;
+		
+		$application_json = apply_filters( 'web_stories_ad_manager_json', $application_json );
```

Comment:
Let's add some docblock for this:

Also, minor suggestion to rename to `$configuration`.

```suggestion
		/**
		 * Filters Google Ad Manager configuration passed to `<amp-story-auto-ads>`.
		 *
		 * @since 1.10.0
		 *
		 * @param array $settings Ad Manager configuration.
		 */
		$configuration = apply_filters( 'web_stories_ad_manager_configuration', $configuration );
```